packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -273,29 +273,31 @@ function* doSendMessage(action: ReturnType<typeof sendMessage>) {
     }
 
     // Optimistically add the message
-    yield* put(
-      addMessage({
-        chatId,
-        message: {
-          sender_user_id: currentUserId,
-          message_id: messageId,
-          message,
-          reactions: [],
-          created_at: dayjs().toISOString()
-        },
-        status: Status.LOADING,
-        isSelfMessage: true
-      })
-    )
+    if (!resend) {
```

Comment:
I wonder if we even need this block - addMessage should dedupe messages with the same messageId, so maybe OK?